apps/wallet/src/ui/app/pages/home/transfer-coin/TransferCoinForm/StepOne.tsx

```diff
@@ -61,51 +75,124 @@ function StepOne({
         isCoinDecimalsLoading,
         validateForm,
     ]);
+
+    const [formattedGas] = useFormatCoin(gasCostEstimation, GAS_TYPE_ARG);
+    const [maxToken, symbol, queryResult] = useFormatCoin(balance, coinType);
```

Comment:
Did you try this with max bigger than 1000? I think we format with 1k not sure if it will have the expected behaviour

Reply:
yes, `100,000,000 SUI`